tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -149,6 +149,24 @@ Feature: full text search
       | spaces           |
 
 
+  Scenario Outline: search version restored files using a tag
+    Given using <dav-path-version> DAV path
+    And user "Alice" has uploaded file with content "hello world" to "test.txt"
+    And user "Alice" has created the following tags for file "test.txt" of the space "Personal":
+      | tag1 |
+    And user "Alice" has uploaded file with content "Namaste nepal" to "test.txt"
+    And user "Alice" has restored version index "1" of file "test.txt"
+    When user "Alice" searches for "Tags:tag1" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result of user "Alice" should contain only these entries:
+      | /test.txt |
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
```

Comment:
can the content be also checked after the search when the version is restored.

Reply:
ya I have already make the PR for that scenario https://github.com/owncloud/ocis/pull/6970